src/Adapters/Laravel/Commands/TestCommand.php

```diff
@@ -66,11 +67,6 @@ public function handle()
             throw new RuntimeException('Running Collision ^5.0 artisan test command requires PHPUnit ^9.0.');
         }
 
-        // @phpstan-ignore-next-line
-        if ((int) \Illuminate\Foundation\Application::VERSION[0] < 8) {
-            throw new RuntimeException('Running Collision ^5.0 artisan test command requires Laravel ^8.0.');
-        }
-
         $options = array_slice($_SERVER['argv'], $this->option('without-tty') ? 3 : 2);
 
         $this->clearEnv();
```

Comment:
@GrahamCampbell Previously, and thanks to you, we added correct support to dotenv 5 in Collision 5. In this pull request, @owenvoke is trying to add support to dotenv 4 (therefore Laravel 7) to Collision 5.

Normally, this code https://github.com/nunomaduro/collision/blob/f9e4f85e2977b1f6c840b243e709132fe8fe0da4/src/Adapters/Laravel/Commands/TestCommand.php#L155 is responsible for cleaning env variables previously loaded. And, thanks to our tests, we know that this is working correctly.

Unfortunately, while using dotenv 4, the following scenario is failing: https://github.com/nunomaduro/collision/runs/874747762?check_suite_focus=true#step:6:65 - One env variable loaded from `.env` is still present (  not correctly cleared ) on the environment when a `.env.testing` file exists.

I know that this can be complicated to understand, so let me know if you have time and motivation to take a look at this.

@owenvoke Let me know if you agree with this description.

Reply:
Yeah, that appears to be the issue. It's a shame we can't just support only DotEnv 5. 🤔

Reply:
Let's see @GrahamCampbell can have a quick fix without taking too much of his time. Otherwise lets just drop the support of dotenv 4.

Reply:
Why can't people just use the v4 with Laravel 7, rather than add Laravel 7 support to v5?

Reply:
I can copy the code over from my previous PR if needed. The good news for Laravel 9 is that I am unlikely to release a new major version of phpdotenv before then, so phpdotenv v5 should be around for a while (so there will be no future hacks to be added for more version support), though so will v4, since I will continue supporting it for Laravel 6 LTS users.